frontend/sky/app/view/dataset/DatasetController.js

```diff
@@ -405,10 +407,12 @@ Ext.define('Sky.view.dataset.DatasetController', {
 
     },
 
-    onActivate: function () {
+    onActivate: function (event) {
         var me = this, coordinate, zoom, aladin,
             visiomatic = me.lookupReference('visiomatic');
 
+        me.showPin = (event.action == 'dblclick');
```

Comment:
@fabionogueira vc está criando um atributo na classe mais não declarou ele, apesar de funcionar corretamente, dificulta depois a leitura do código pq tem que procurar onde está atribuindo esse valor. vc pode adicionar novos atributos dentro do atributo config: { ..... } da classe dessa forma o componente já vai gerar pra vc um metodo set e um get para seu atributo. assim qualquer um que for debugar ou usar essa classe vai saber da existencia deste atributo. independente se ele foi usado ou não.